moveit_ros/hybrid_planning/test/config/demo_controller.yaml

```diff
@@ -0,0 +1,28 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 500  # Hz
```

Comment:
Changing this (to 100) has strange effects on the demo, which I cannot explain. Sometimes the robot moves a bit and then stops

Reply:
I've experienced the same thing and it started when I switched from fake joint to ros2_control based fake joint. This fixed it for me. Initially, I thought it might be related to an insufficient frequency of joint state updates but maybe I need to take a look at the connection between the local planner - and underlying controller frequency

Reply:
Is this an int type and not a floating point?